.github/workflows/docker-ci.yml

```diff
@@ -68,6 +69,13 @@ jobs:
           username: ${{ github.repository_owner }}+github
           password: ${{ secrets.BOT_QUAY_IO }}
 
+      - name: Login to docker.io Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: docker.io
+          username: ${{ secrets.USER_DOCKER_IO }}
+          password: ${{ secrets.BOT_DOCKER_IO }}
```

Comment:
Please, could you confirm these secrets have been populated in the repository/organization?